README.md

```diff
@@ -136,4 +136,4 @@ log.fatal('Message', 'Details');
 
 ## Socials
 
-Join our discord community [here](https://discord.gg/jZQs6Wu)
+Join our discord community click [here](https://discord.gg/jZQs6Wu)
```

Comment:
Can this be rephrased to `Click [here](https://discord.gg/jZQs6Wu) to join our discord community` ?

Reply:
The whole sentence could be a link. I'm not sure what do you think?

Reply:
Yah that would be better :+1:
`[Join our discord community-!](https://discord.gg/jZQs6Wu)`